### DISCUSSION TITLE
想请教一下在cocos creator中如何接入PAG

### DISCUSSION THREAD

**Discussion by YoYoHu0527** [Tencent/libpag#2070] [Q&A] [answered]
我使用的是Cocos Creator 3.72版本 ts语言开发的。现在我希望能在项目中使用PAG动画，项目发布是在web端运行的

**Comment by kevingpqi123** [comment, marked as answer]
引擎层面的使用需要深入了解 pag 源码中平台层面的接口封装，由于我们自身没有这样的业务场景，可能需要自己去探索
